src/styles.css

```diff
@@ -0,0 +1,204 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+* {
+  color: black;
+  background-color: #f5f6fa;
+}
+
+.App {
+  /* overflow-x: hidden; */
+}
+
+.my-navbar {
+  margin: 0 0 38px;
+  padding: 23px 23px 23px 100px;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+}
+
+.fa-user {
+  color: #0290ff;
+}
+
+.oval {
+  margin: 0 0 0 4px;
+  padding: 14px;
+  border: solid 1px #e8e8e8;
+  border-radius: 50%;
+  background-color: #fff;
+}
+
+.logo {
+  display: flex;
+  background-color: #fff;
+}
+
+.margin-75 {
+  /* width: 75%; */
+  margin-left: 100px;
+  margin-right: 100px;
+}
+
+.header-book {
+  background-color: #fff;
+  margin: 18px 31px 1px 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 13px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.header-categories {
+  background-color: #fff;
+  margin: 18px 0 11px 11px;
+  font-family: Montserrat, sans-serif;
+  font-size: 13px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```

Comment:
These specific properties have been repeated for most of your CSS rules. It is better to have them in one rule with a selector combining all the affected elements. This would make your code more maintainable. Check out this ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)